docs/detections/api/rules/rules-api-export.asciidoc

```diff
@@ -1,14 +1,18 @@
 [[rules-api-export]]
 === Export rules
 
-Exports rules to an `.ndjson` file. Please note that you cannot export prebuilt rules, but they are available at https://github.com/elastic/detection-rules/tree/main/rules/.
+Exports rules to an `.ndjson` file. The following configuration items are also included in the `.ndjson` file:
 
-NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+* Actions
+* Exception lists
+
+Please note that you cannot export prebuilt rules, but they are available at https://github.com/elastic/detection-rules/tree/main/rules/.
 
+NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
 [IMPORTANT]
 =================
-Detection rule actions are included in the exported file, but the connectors used by the actions are not included. 
+Although detection rule actions are included in the exported file, the connectors used by the actions are not included. 
```

Comment:
Might also be important to note that what we refer to as "large value lists/value lists" are not included in the export/import. Value lists are lists that can be referenced within an exception item. Value lists will need to be imported/exported separately.

Reply:
@yctercero Is there a recommended way to export/import value lists? For example, in the next line we suggest using the Saved Objects UI or APIs to export/import connectors, so can we say the same thing about value lists?

Reply:
<img width="1732" alt="Screen Shot 2022-01-25 at 9 44 12 AM" src="https://user-images.githubusercontent.com/10927944/151030645-f0d1ac04-dadd-40d9-a87a-c8c3325017ec.png">

These lists can be imported/exported on the rules management page.